### PR TITLE
Removed hardcoded dot prefix from the transmission category [migration 81]

### PR DESCRIPTION
**Pull request by Taloth**
With migration to add the dot to existing configured Transmission Settings so we don't break existing users.

### CODE REVIEW

**Comment by kayone** [comment]
needs migration tests

File: src/NzbDrone.Core/Datastore/Migration/081_move_dot_prefix_to_transmission_category.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                {
+                    while (reader.Read())
+                    {
+                        var id = reader.GetInt32(0);
+                        var settingsJson = reader.GetString(1);
+
+                        var settings = Json.Deserialize<Dictionary<string, object>>(settingsJson);
+
+                        var tvCategory = settings.GetValueOrDefault("tvCategory") as string;
+                        if (!tvCategory.IsNullOrWhiteSpace())
```
Comment (kayone) [inline src/NzbDrone.Core/Datastore/Migration/081_move_dot_prefix_to_transmission_category.cs]: `IsNotNullOrWhiteSpace `

**Comment by Taloth** [comment]
There you go, migration test.

**Comment by kayone** [comment]
are there any indication that someone might be launching one of these?

**Pull request merged**